From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 16 Mar 2026 10:31:00 +0800
Subject: input-err

# DP: Define PGRP_PIPE to avoid race condition.

# DP: Define PGRP_PIPE to avoid race condition.
---
 input.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/input.c b/input.c
index 9995b94..f715dca 100644
--- a/input.c
+++ b/input.c
@@ -520,7 +520,7 @@ b_fill_buffer (BUFFERED_STREAM *bp)
       if (nr == 0)
 	bp->b_flag |= B_EOF;
       else
-	bp->b_flag |= B_ERROR;
+	fatal_error("error reading input file: %s", strerror(errno));
       return (EOF);
     }
 
